frontend/ui/widget/opdsbrowser.lua

```diff
@@ -504,7 +504,7 @@ function OPDSBrowser:downloadFile(item, format, remote_url)
     logger.dbg("downloading file", local_path, "from", remote_url)
 
     local parsed = url.parse(remote_url)
-    http.TIMEOUT, https.TIMEOUT = 10, 10
+    http.TIMEOUT, https.TIMEOUT = 20, 20
     local httpRequest = parsed.scheme == 'http' and http.request or https.request
     local _, c, _ = httpRequest{
```

Comment:
Would it make more sense to change this to something like `__` than to add a `local _ = require("gettext")`?